README-Azure.md

```diff
@@ -78,7 +66,8 @@ public class BlobController : Controller
         {
             // If you want to have get a blob on behalf of the app itself.
             _tokenCredential.Options.RequestAppToken = true;
-            var containerClient = _blobServiceClient.GetBlobContainerClient(containerName);
+            BlobContainerClient containerClient = new BlobContainerClient(new Uri($"https://storageaccountname.blob.core.windows.net/{containername}"), _microsoftIdentityTokenCredential);
```

Comment:
This doesn't follow azure SDK guidance: https://devblogs.microsoft.com/azure-sdk/lifetime-management-and-thread-safety-guarantees-of-azure-sdk-net-clients/

**The main rule of Azure SDK client lifetime management is: treat clients as singletons.**

This will create a client per controller, and is going to be a lot of overhead.